css/global.css

```diff
@@ -12,3 +12,13 @@ aside > section.artdeco-card.ember-view.relative.break-words.pb3.mt2,
 li.social-details-social-counts__reactions {
   visibility: hidden !important;
 }
+
+/* hides follower count visible when looking at your own posts and activity */
+a[href="/mynetwork/network-manager/people-follow/followers/"] {
```

Comment:
[This article ](https://www.w3schools.com/css/css_attribute_selectors.asp)details more about attribute selectors in CSS. This line looks for an anchor tag with the href value of "/mynetwork/network-manager/people-follow/followers/", which takes users to their list of followers, and hides the element from the DOM.